src/researchhub_case/migrations/0008_authorclaimcase_claim_context.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2 on 2022-02-11 20:24
```

Comment:
A bunch of migrations because I went back and forth on a few fields. The net change is a new fields: `target_paper` and `target_author_name`